src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Memcached/EnyimMemcachedCoreWrapper.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NewRelic.Agent.Api;
+using NewRelic.Agent.Extensions.Parsing;
+using NewRelic.Agent.Extensions.Providers.Wrapper;
+
+namespace NewRelic.Providers.Wrapper.Memcached
+{
+    public class EnyimMemcachedCoreWrapper : IWrapper
+    {
+        private const string ModelName = "cache";
+
+        public string[] WrapperNames = new string[] { "EnyimMemcachedCoreWrapper" };
+
+        public bool IsTransactionRequired => true;
+
+        public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
+        {
+            var canWrap = WrapperNames.Contains(methodInfo.RequestedWrapperName, StringComparer.OrdinalIgnoreCase);
```

Comment:
Might be a little simpler not to use a list here, since there's only one wrapper name.

Reply:
Fixed.